tests/unit_tests/test_league_rater.py

```diff
@@ -0,0 +1,282 @@
+import pytest
+
+from service import config
+from service.league_service.league_rater import LeagueRater
+from service.league_service.typedefs import (
+    GameOutcome,
+    League,
+    LeagueDivision,
+    LeagueScore,
+)
+
+
+@pytest.fixture
+def unplaced_player_score():
+    return LeagueScore(division_id=None, score=None, game_count=10)
+
+
+def test_new_score_victory_no_boost(example_league):
+    current_score = LeagueScore(division_id=2, score=5, game_count=30)
+    player_rating = (180.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score + config.SCORE_GAIN
+
+
+def test_new_score_victory_boost(example_league):
+    current_score = LeagueScore(division_id=2, score=5, game_count=30)
+    player_rating = (1800.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score + config.SCORE_GAIN + config.POSITIVE_BOOST
+
+
+def test_new_score_defeat_no_boost(example_league):
+    current_score = LeagueScore(division_id=2, score=5, game_count=30)
+    player_rating = (180.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.DEFEAT,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score - config.SCORE_GAIN
+
+
+def test_new_score_defeat_boost(example_league):
+    current_score = LeagueScore(division_id=2, score=5, game_count=30)
+    player_rating = (60.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.DEFEAT,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score - config.SCORE_GAIN - config.NEGATIVE_BOOST
+
```

Comment:
There should also be two more tests checking that 
1. if you would have a victory boost but lose, you still lose only one point
2. if you would have a nonzero `NEGATIVE_BOOST` but win you still gain a point.